test/distributed_commands_on_value_types_test.rb

```diff
@@ -6,6 +6,14 @@ class TestDistributedCommandsOnValueTypes < Test::Unit::TestCase
   include Helper::Distributed
   include Lint::ValueTypes
 
+  def test_move
+    assert_raise(Redis::Distributed::CannotDistribute) { super }
```

Comment:
It is raised from [SELECT](https://github.com/redis/redis-rb/blob/v4.1.0/test/lint/value_types.rb#L108).